README.md

```diff
@@ -1,2 +1,44 @@
-# flask
-Starting a basic Python Flask server
+# flask - project
+Agenda:
```

Comment:
recommendation: Building a static Web page with Python Flask Web Framework in Google Cloud Platform using Terraform Orchestration tool